### PR TITLE
iev17 Change business unit

### PR DESCRIPTION
**Pull request by Brandonz07**

### CODE REVIEW

**Review by novadeveloper14 — APPROVED** [review]

File: src/components/inputs/BusinessUnitChange/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <StyledUl>
+          {clients.map((client, index) => (
+            <Link key={client.id} to="#" onClick={() => handleLogoClick(client)}>
+              <StyledItem>
+                <StyledImg src={client.logo} alt={client.name} />
+                {selectedClient === client.sigla && (
+                <Icon
+                  icon={<MdCheck />}
+                  appearance="primary"
+                  size="20px"
```
Comment (carin91) [inline src/components/inputs/BusinessUnitChange/index.tsx:43]: Hold consistency between icons sizes.
image
Verify the correct size, however i found a 24px for MdInfoOutline icon

File: src/components/inputs/BusinessUnitChange/index.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  return (
+    <StyledContainer>
+      <Stack width="220px">
+        <StyledUl>
+          {clients.map((client, index) => (
+            <Link key={client.id} to="#" onClick={() => handleLogoClick(client)}>
+              <StyledItem>
+                <StyledImg src={client.logo} alt={client.name} />
+                {selectedClient === client.sigla && (
+                <Icon
```
Comment (carin91) [inline src/components/inputs/BusinessUnitChange/index.tsx:40]: The check icon should be rendered before the first click based on the selected business unit.
image

File: src/components/layout/AppPage/index.tsx [hunk 3]
```diff
@@ -78,6 +86,15 @@ function AppPage() {
             client={user.company}
           />
         </StyledHeaderContainer>
+        <StyledCollapseIcon $collapse={collapse} onClick={() =>  setCollapse(!collapse)} $isTablet={isTablet}>
+            <Icon
```
Comment (carin91) [inline src/components/layout/AppPage/index.tsx:90]: Hold consistency between icons sizes.
image
Verify the correct size, however i found a 24px for MdInfoOutline icon

Comment (carin91) [inline src/components/layout/AppPage/index.tsx:90]: The icon position seems invalid, it is not centered with the business unit logo
image

File: src/components/layout/AppPage/index.tsx [hunk 4]
```diff
@@ -78,6 +86,15 @@ function AppPage() {
             client={user.company}
           />
         </StyledHeaderContainer>
+        <StyledCollapseIcon $collapse={collapse} onClick={() =>  setCollapse(!collapse)} $isTablet={isTablet}>
```
Comment (carin91) [inline src/components/layout/AppPage/index.tsx:89]: (1) The dropdown of the business unit should close when the user clicks outside of it.

File: src/components/inputs/BusinessUnitChange/index.tsx [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    localStorage.setItem('selectedClientSigla', selectedClient)
+  }, [selectedClient])
+
+  const handleLogoClick = (client: IClient) => {
+    handleClientChange(client);
+    setSelectedClient(client.sigla);
+  };
+
+  return (
+    <StyledContainer>
```
Comment (carin91) [inline src/components/inputs/BusinessUnitChange/index.tsx:32]: Adjust the responsive behavior, the position of the dropdown seems invalid considering the view of the searcher, add to it, the logos and icons seem bigger than the selected logo/icon 
image

**Review by carin91 — CHANGES_REQUESTED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**